hashing/src/chunk_with_proof.rs

```diff
@@ -73,13 +73,11 @@ impl FromBytes for ChunkWithProof {
 
 impl ChunkWithProof {
     #[cfg(test)]
-    pub const CHUNK_SIZE: usize = 1;
+    pub const CHUNK_SIZE: usize = 10;
```

Comment:
That could make many tests extremely slow once we actually use it for the trie.
I'm not sure what would be a good compromise; `1_024` maybe?

Reply:
Ah, sorry, I think the `node` tests will actually use `hashing` without the `testing` flag, so it should be fine.